setup.py

```diff
@@ -23,11 +23,13 @@
     install_requires=[
         "rich-argparse==1.0.0",
         "rich==13.0.1",
-        "mistune==0.8.4",
         "chardet==5.1.0",
         "requests==2.31.0",
         "PyYAML==6.0.1",
         "gitignorefile==1.1.2",
+        "mistune==3.0.1",
+        "tortilla==0.5.0",
```

Comment:
Hmm, not sure why tortilla is listed here. I moved away from it a few versions ago.